### ISSUE TITLE
Preview stretches for a moment while switching front/back

### ISSUE THREAD

**Issue by dhruv2295** [natario1/CameraView#466]
### Describe the bug

- CameraView version: *2.0.0-beta04*
- Reproducible in official demo app: *yes*
- Device / Android version: *Pixel 3, API 28*

### To Reproduce
Steps to reproduce the behavior, possibly in the demo app:
1. Select Video Mode
2. Set width and height to match_parent
3. Click on toggle camera button

### Expected behavior
While switching the preview screen stretches for a second

**Comment by natario1** [comment]
I'm not sure this can be fixed... But will take a look.

**Comment by MoustafaElsaghier** [comment]
Mr @natario1 any updates on this issue?

**Comment by jlertele** [comment]
I have the same problem...

**Comment by wirteEable** [comment]
+1

**Comment by vaidik1011** [comment]
+1

**Comment by vaidik1011** [comment]
+1

**Comment by vaidik1011** [comment]
It will probably not stretches if you give camera view ratio of 4:3. maybe it's not the solution but you can overcome that problem using it.

**Comment by SweetD3v** [comment]
Yes, It won't stretch if you keep the same aspect ratio on either of the cameras.

**Comment by xiaoyaomeng** [comment]
I have the same problem...
